### PR TITLE
Slider: Implement range slider (two thumb)

### PR DESCRIPTION
**Pull request by akashyeole**
By creating this pull request you agree to the terms in CONTRIBUTING.md.
https://github.com/Infineon/.github/blob/master/CONTRIBUTING.md
--- DO NOT DELETE ANYTHING ABOVE THIS LINE ---

CONTRIBUTING.md also tells you what to expect in the PR process.

Description
Implemented 2 thumb range slider.
Fixed wrong percentage calculation bug.

Related Issue
#1147
#1111 

<details>
  <summary>📦 Published PR as canary version: <code>21.2.0--canary.1157.50ef9a5424457d76266590dd1d6067b5bd22cb15.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @infineon/infineon-design-system-stencil@21.2.0--canary.1157.50ef9a5424457d76266590dd1d6067b5bd22cb15.0
  # or 
  yarn add @infineon/infineon-design-system-stencil@21.2.0--canary.1157.50ef9a5424457d76266590dd1d6067b5bd22cb15.0
  ```
</details>

### CODE REVIEW

File: packages/components/src/components/slider/slider.tsx [hunk 1]
```diff
@@ -9,23 +9,77 @@ export class IfxSlider {
...
+      }else{
+        this.minInputRef.value = this.maxInputRef.value;
+      }
+    }
+    if(target.id === 'max-slider') {
+      this.internalMaxValue = parseInt(this.maxInputRef.value);
+    } else {
+      this.internalMinValue = parseInt(this.minInputRef.value);
+    }
+    this.ifxChange.emit({minVal: this.internalMaxValue, maxVal: this.internalMinValue});
```
Comment (tishoyanchev) [inline packages/components/src/components/slider/slider.tsx:69]: maxValue is assigned to minVal, and minValue is assigned to maxValue.

**Review by tishoyanchev — COMMENTED** [review]
- Rename `type='range'` with `type='double'`, since the slider is a range element by default. 
- Rename `min-value` and `max-value` props to `min-value-handle` and `max-value-handle` for clarity.
- changes are not reflected in the example apps

**Comment by verena-ifx** [comment]
it seems, like the percentage value is not calculated correctly:
image

**Comment by akashyeole** [comment]
> it seems, like the percentage value is not calculated correctly:
@verena-ifx 
This is related to #1111 issue, I will resolve it in this PR

**Comment by verena-ifx** [comment]
I merged master into it, to resolve some merge conflicts

**Comment by akashyeole** [comment]
>The percentage value still needs to be recalculated when the max/min values change

Hi @verena-ifx , can you please check again, it is working fine on my side:

image

**Pull request merged**